clayui.com/gatsby/onCreateNode.js

```diff
@@ -78,7 +78,10 @@ module.exports = exports.onCreateNode = ({actions, getNode, node}) => {
 						'@clayui/',
 						''
 					)}.html`;
-				} else {
+				} else if (
+					!relativePath.includes('api-') ||
+					!relativePath.includes('markup-')
```

Comment:
Instead of adding the prefix, you could use `mainTabURL` in the conditional. Since everyone who is as a tab will have `mainTabURL`.

Reply:
@matuzalemsteles good idea!